### ISSUE TITLE
Auto Nest .js and .map files under .ts files

### ISSUE THREAD

**Issue by daryllabar** [madskristensen/FileNesting#83]
### Installed product versions
- Visual Studio: 2017 Enterprise
- This extension : 2.6.67

### Description
I'm using a (VS2017) website (Without a csproj) and it is nesting js.map files under the js files, but I'm using TypeScript and would prefer it to nest both the js and map files under the ts file.  Is there a way to get that to work?  I've installed through web essentials if that matters

**Comment by ltlombardi** [comment]
Hey. the website doesn't have a csproj, and because of that, you can't include or remove from the project, is that it? Because if you can, you should remove / not include the .js and .map.

I would recommend creating a simple empty project and adding your website in it (test the templates to see which serves you better), so you could work with your website. That way your problem is solved.

**Comment by daryllabar** [comment]
@ltlombardi I don't believe that solution works if others on my team are using VSCode...

**Comment by ltlombardi** [comment]
I particularly don't use VSCode, but I just opened a .net 4.6 project in VSCode and it worked as a simple IDE, without compiler. I think it's worth the try, because seeing .map and .js must be making you crazy. Good luck
